app/services/analytics.rb

```diff
@@ -10,6 +10,7 @@ def track_event(event, attributes = {})
     analytics_hash = {
       event_properties: attributes.except(:user_id),
       user_id: attributes[:user_id] || user.uuid,
+      locale: I18n.locale,
```

Comment:
have we confirmed this is set correctly for the client-sent analytics? AFAIK those don't send to `/fr/analytics` so there's no URL param to guess from, in case users switch locales manually but don't switch their HTTP accept locale: https://github.com/18F/identity-idp/blob/main/app/services/locale_chooser.rb#L9-L11